### PR TITLE
guix: swap `moreutils` for just `sponge`

### PR DESCRIPTION
**Pull request by fanquake**
Switch to building the only `moreutils` utility we actually need (`sponge`). This results in having less unused stuff in the Guix environment (i.e all the other `moreutils` utilities), and, the dependency graph is simplified. i.e we no-longer have a dependency on `perl`, `docbook` etc, for this package.

Current `moreutils` dependency graph:
moreutils

In the Guix env, `chronic`, `combine`, `errno`, `ifdata`, `ifne`, `isutf8`, `lckdo`, `mispipe`, `parallel`, `pee`, `ts`, `vidir`, `vipe` & `zrun` (plus their `*.real` variants) are removed.

### CODE REVIEW

**Comment by hebasto** [comment]
Concept ACK.

**Comment by TheCharlatan** [comment]
Nice, Concept ACK

**Comment by TheCharlatan** [comment]
Checked the patch against the original moreutils package, and reproduced the stripped-down package. I think grabbing the specific tools we need from these kind of packages is a good thing.

Guix builds (aarch64):
```
74a02707614d7fc7ed460148b37e90c126d09c2e9b2962765de14d267a3f22d3  guix-build-c4ee9b8aa078/output/aarch64-linux-gnu/SHA256SUMS.part
263f079da730485eb583995c1fc6304b7ca957a668fd5aac8f879bc59746dc67  guix-build-c4ee9b8aa078/output/aarch64-linux-gnu/bitcoin-c4ee9b8aa078-aarch64-linux-gnu-debug.tar.gz
745a9bfdd9550cf9abf80415c4efe818446ea05e9c4f659f643669c3dcb49893  guix-build-c4ee9b8aa078/output/aarch64-linux-gnu/bitcoin-c4ee9b8aa078-aarch64-linux-gnu.tar.gz
5810b1efc04172fec17bab60082fec045c4a23c93050baeb391ac2f2a73c6e63  guix-build-c4ee9b8aa078/output/arm-linux-gnueabihf/SHA256SUMS.part
501eab289ad3e2eec030425ee3fbede73c7ec3f23b42377e330fec8af32c1de6  guix-build-c4ee9b8aa078/output/arm-linux-gnueabihf/bitcoin-c4ee9b8aa078-arm-linux-gnueabihf-debug.tar.gz
827edd55ebf01fe26f6ae2caa2cefac048f2a697bfdd22728f64fb61ab30d660  guix-build-c4ee9b8aa078/output/arm-linux-gnueabihf/bitcoin-c4ee9b8aa078-arm-linux-gnueabihf.tar.gz
5eb22cc353375cc8e37f0e7dfd897029d0ae227730ecd0c0b8cc23434e609026  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/SHA256SUMS.part
d16b7d032ce11110e68306d303e6c712bafbdabb86541d29c5a3c88f62c8600d  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/bitcoin-c4ee9b8aa078-arm64-apple-darwin-unsigned.tar.gz
26faf4775d2ff65260c6ebd7ccd76d6fd69633d3658b7a3d6c2aa3f07a975204  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/bitcoin-c4ee9b8aa078-arm64-apple-darwin-unsigned.zip
38767a02f670f1e3ddc60be7f442e9b555dcdf3e6e1df101b9e607b958b48e9c  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/bitcoin-c4ee9b8aa078-arm64-apple-darwin.tar.gz
5ee0ba02fafd22a488ad14ca6c54485e9756724de9076122e02a460a9dc966d8  guix-build-c4ee9b8aa078/output/dist-archive/bitcoin-c4ee9b8aa078.tar.gz
80e1da69b4a56313660c69b987c4c0d8a66f8635e8f3c2a007f15cfeeae87c73  guix-build-c4ee9b8aa078/output/powerpc64-linux-gnu/SHA256SUMS.part
eb731a4f1bdd32b7801cdda3de20db6fb534c42249cd4b2207f8919780223e48  guix-build-c4ee9b8aa078/output/powerpc64-linux-gnu/bitcoin-c4ee9b8aa078-powerpc64-linux-gnu-debug.tar.gz
0378720813c837f4ca5ba345630b9b5447a1b00c1634985024e6898ac551f875  guix-build-c4ee9b8aa078/output/powerpc64-linux-gnu/bitcoin-c4ee9b8aa078-powerpc64-linux-gnu.tar.gz
f97e69bd741f2a7c38a3022acd84dd331f5be11230eb502bcbb936d84ac85ca9  guix-build-c4ee9b8aa078/output/riscv64-linux-gnu/SHA256SUMS.part
07a7d1edafb77d43644530040ccb22865d059ffd0dd1cbe3bcba08a9cab5f6c0  guix-build-c4ee9b8aa078/output/riscv64-linux-gnu/bitcoin-c4ee9b8aa078-riscv64-linux-gnu-debug.tar.gz
876191186998d63e549bb6679c848ecbe0d7296092198e85d31983035734f3d0  guix-build-c4ee9b8aa078/output/riscv64-linux-gnu/bitcoin-c4ee9b8aa078-riscv64-linux-gnu.tar.gz
213e98d14ff34079376a06188c7e2e328cfdf2a8a1145b284b804ed97e03c235  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/SHA256SUMS.part
46cb71117952c371ff152cd686554522b55ca8f17336e7dd64bb9fc223fdf4ae  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/bitcoin-c4ee9b8aa078-x86_64-apple-darwin-unsigned.tar.gz
d2d029ce8886fdd1701e5acb7ffd54c94fba9f77337517e2bc87f306a4483e89  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/bitcoin-c4ee9b8aa078-x86_64-apple-darwin-unsigned.zip
2592b4df5270b01baa7ef929d7b98d2512262fcac2a5d773bb1befb36c05b347  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/bitcoin-c4ee9b8aa078-x86_64-apple-darwin.tar.gz
2a441abe210364b95e440ac374031313793ee95286c251079a653035c8f78654  guix-build-c4ee9b8aa078/output/x86_64-linux-gnu/SHA256SUMS.part
69a8acf7797cfb0ae3fa73c82193454a8a71827f8aaa530d1a4dfaef8223bfbe  guix-build-c4ee9b8aa078/output/x86_64-linux-gnu/bitcoin-c4ee9b8aa078-x86_64-linux-gnu-debug.tar.gz
293a245727edf76296f24fbbb853422ce18b924690db586edc45660e3d4eae8a  guix-build-c4ee9b8aa078/output/x86_64-linux-gnu/bitcoin-c4ee9b8aa078-x86_64-linux-gnu.tar.gz
4a8dcedcedb26d19e5d556c4f38ba51cb45be24fd15efbb2808355723506ea97  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/SHA256SUMS.part
ea784b1d8a350a538ddf9258c9f4ba764ac088232919e8ddd8a2fa5a1740e746  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64-debug.zip
1d00d2b90fe47f52a6ced4daca8e6e37f16c8215540150214dbefbdbf375ef5a  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64-setup-unsigned.exe
cb6e1533402ddf487cc71481926768a773868701d66f0f5655806b7f6160a50e  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64-unsigned.tar.gz
2313eafb2af4d2f9292cbe2537a8611a98f98d07c98706005f7c6aa759279ac7  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64.zip
```

**Comment by kevkevinpal** [comment]
Concept ACK

**Review by hebasto — COMMENTED** [review]
My Guix build:
```
riscv64
74a02707614d7fc7ed460148b37e90c126d09c2e9b2962765de14d267a3f22d3  guix-build-c4ee9b8aa078/output/aarch64-linux-gnu/SHA256SUMS.part
263f079da730485eb583995c1fc6304b7ca957a668fd5aac8f879bc59746dc67  guix-build-c4ee9b8aa078/output/aarch64-linux-gnu/bitcoin-c4ee9b8aa078-aarch64-linux-gnu-debug.tar.gz
745a9bfdd9550cf9abf80415c4efe818446ea05e9c4f659f643669c3dcb49893  guix-build-c4ee9b8aa078/output/aarch64-linux-gnu/bitcoin-c4ee9b8aa078-aarch64-linux-gnu.tar.gz
5810b1efc04172fec17bab60082fec045c4a23c93050baeb391ac2f2a73c6e63  guix-build-c4ee9b8aa078/output/arm-linux-gnueabihf/SHA256SUMS.part
501eab289ad3e2eec030425ee3fbede73c7ec3f23b42377e330fec8af32c1de6  guix-build-c4ee9b8aa078/output/arm-linux-gnueabihf/bitcoin-c4ee9b8aa078-arm-linux-gnueabihf-debug.tar.gz
827edd55ebf01fe26f6ae2caa2cefac048f2a697bfdd22728f64fb61ab30d660  guix-build-c4ee9b8aa078/output/arm-linux-gnueabihf/bitcoin-c4ee9b8aa078-arm-linux-gnueabihf.tar.gz
5eb22cc353375cc8e37f0e7dfd897029d0ae227730ecd0c0b8cc23434e609026  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/SHA256SUMS.part
d16b7d032ce11110e68306d303e6c712bafbdabb86541d29c5a3c88f62c8600d  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/bitcoin-c4ee9b8aa078-arm64-apple-darwin-unsigned.tar.gz
26faf4775d2ff65260c6ebd7ccd76d6fd69633d3658b7a3d6c2aa3f07a975204  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/bitcoin-c4ee9b8aa078-arm64-apple-darwin-unsigned.zip
38767a02f670f1e3ddc60be7f442e9b555dcdf3e6e1df101b9e607b958b48e9c  guix-build-c4ee9b8aa078/output/arm64-apple-darwin/bitcoin-c4ee9b8aa078-arm64-apple-darwin.tar.gz
5ee0ba02fafd22a488ad14ca6c54485e9756724de9076122e02a460a9dc966d8  guix-build-c4ee9b8aa078/output/dist-archive/bitcoin-c4ee9b8aa078.tar.gz
80e1da69b4a56313660c69b987c4c0d8a66f8635e8f3c2a007f15cfeeae87c73  guix-build-c4ee9b8aa078/output/powerpc64-linux-gnu/SHA256SUMS.part
eb731a4f1bdd32b7801cdda3de20db6fb534c42249cd4b2207f8919780223e48  guix-build-c4ee9b8aa078/output/powerpc64-linux-gnu/bitcoin-c4ee9b8aa078-powerpc64-linux-gnu-debug.tar.gz
0378720813c837f4ca5ba345630b9b5447a1b00c1634985024e6898ac551f875  guix-build-c4ee9b8aa078/output/powerpc64-linux-gnu/bitcoin-c4ee9b8aa078-powerpc64-linux-gnu.tar.gz
f97e69bd741f2a7c38a3022acd84dd331f5be11230eb502bcbb936d84ac85ca9  guix-build-c4ee9b8aa078/output/riscv64-linux-gnu/SHA256SUMS.part
07a7d1edafb77d43644530040ccb22865d059ffd0dd1cbe3bcba08a9cab5f6c0  guix-build-c4ee9b8aa078/output/riscv64-linux-gnu/bitcoin-c4ee9b8aa078-riscv64-linux-gnu-debug.tar.gz
876191186998d63e549bb6679c848ecbe0d7296092198e85d31983035734f3d0  guix-build-c4ee9b8aa078/output/riscv64-linux-gnu/bitcoin-c4ee9b8aa078-riscv64-linux-gnu.tar.gz
213e98d14ff34079376a06188c7e2e328cfdf2a8a1145b284b804ed97e03c235  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/SHA256SUMS.part
46cb71117952c371ff152cd686554522b55ca8f17336e7dd64bb9fc223fdf4ae  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/bitcoin-c4ee9b8aa078-x86_64-apple-darwin-unsigned.tar.gz
d2d029ce8886fdd1701e5acb7ffd54c94fba9f77337517e2bc87f306a4483e89  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/bitcoin-c4ee9b8aa078-x86_64-apple-darwin-unsigned.zip
2592b4df5270b01baa7ef929d7b98d2512262fcac2a5d773bb1befb36c05b347  guix-build-c4ee9b8aa078/output/x86_64-apple-darwin/bitcoin-c4ee9b8aa078-x86_64-apple-darwin.tar.gz
2a441abe210364b95e440ac374031313793ee95286c251079a653035c8f78654  guix-build-c4ee9b8aa078/output/x86_64-linux-gnu/SHA256SUMS.part
69a8acf7797cfb0ae3fa73c82193454a8a71827f8aaa530d1a4dfaef8223bfbe  guix-build-c4ee9b8aa078/output/x86_64-linux-gnu/bitcoin-c4ee9b8aa078-x86_64-linux-gnu-debug.tar.gz
293a245727edf76296f24fbbb853422ce18b924690db586edc45660e3d4eae8a  guix-build-c4ee9b8aa078/output/x86_64-linux-gnu/bitcoin-c4ee9b8aa078-x86_64-linux-gnu.tar.gz
4a8dcedcedb26d19e5d556c4f38ba51cb45be24fd15efbb2808355723506ea97  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/SHA256SUMS.part
ea784b1d8a350a538ddf9258c9f4ba764ac088232919e8ddd8a2fa5a1740e746  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64-debug.zip
1d00d2b90fe47f52a6ced4daca8e6e37f16c8215540150214dbefbdbf375ef5a  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64-setup-unsigned.exe
cb6e1533402ddf487cc71481926768a773868701d66f0f5655806b7f6160a50e  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64-unsigned.tar.gz
2313eafb2af4d2f9292cbe2537a8611a98f98d07c98706005f7c6aa759279ac7  guix-build-c4ee9b8aa078/output/x86_64-w64-mingw32/bitcoin-c4ee9b8aa078-win64.zip
```

**Review by sedited — APPROVED** [review]
ACK c4ee9b8aa07824610e1de9c70a4b938844c5ff53

File: contrib/guix/manifest.scm [hunk 1]
```diff
@@ -487,6 +487,38 @@ inspecting signatures in Mach-O binaries.")
...
+           #~(modify-phases %standard-phases
+               (delete 'configure)
+               (replace 'install
+                (lambda* (#:key outputs #:allow-other-keys)
+                  (let ((bin (string-append (assoc-ref outputs "out") "/bin")))
+                  (install-file "sponge" bin)))))
+           #:make-flags
+           #~(list "sponge" (string-append "CC=" #$(cc-for-target)))))
+    (inputs
+     (list bash-minimal))
```
[the remark this answers is not in the document]
Comment (fanquake) [reply contrib/guix/manifest.scm:516]: Bash is implied/available, so we are able to drop this.

**Review by hebasto — APPROVED** [review]
ACK e8f50c5debe2b866e59872dbbf22db0fd592ec04.

My Guix build:
```
aarch64
c3d79b98874ba44dbe6550fd18cdd78443ee1b4d0594dc1cc2e6b423dcded117  guix-build-e8f50c5debe2/output/aarch64-linux-gnu/SHA256SUMS.part
159a09c6ed25d498a48f3e3d53dd4c77b61c2ecbd824661593316030678ee5b0  guix-build-e8f50c5debe2/output/aarch64-linux-gnu/bitcoin-e8f50c5debe2-aarch64-linux-gnu-debug.tar.gz
34cd42b0a3574085c173073bbb12cbb19db2169efadab752335071eb897c242e  guix-build-e8f50c5debe2/output/aarch64-linux-gnu/bitcoin-e8f50c5debe2-aarch64-linux-gnu.tar.gz
0a62bc6b870d6dc36a4926ebcd546f001363ed3f5453ce80e20d929313e1a1ee  guix-build-e8f50c5debe2/output/arm-linux-gnueabihf/SHA256SUMS.part
374b3aeb02c3d9b0324ecce254ccdaf172e9711bf2ca5a821178cb57440f076c  guix-build-e8f50c5debe2/output/arm-linux-gnueabihf/bitcoin-e8f50c5debe2-arm-linux-gnueabihf-debug.tar.gz
dd088f2a79b31041f895ce3791ebaf4a9b6d79cefea79ec99dfbe1fee3adae87  guix-build-e8f50c5debe2/output/arm-linux-gnueabihf/bitcoin-e8f50c5debe2-arm-linux-gnueabihf.tar.gz
735bc9fd04d401527adc298d89b42a46c3d3a5dd80dea9a14ae00eefc6c1c5db  guix-build-e8f50c5debe2/output/arm64-apple-darwin/SHA256SUMS.part
4a478126accb0817e02c1009d0fa92dc2539fbd84b75fbc30d2eb97ae82c66a8  guix-build-e8f50c5debe2/output/arm64-apple-darwin/bitcoin-e8f50c5debe2-arm64-apple-darwin-unsigned.tar.gz
783ef9e80244cc9052c54bc4ece78d736f3109dc24f5fcef46c6b55a995398f4  guix-build-e8f50c5debe2/output/arm64-apple-darwin/bitcoin-e8f50c5debe2-arm64-apple-darwin-unsigned.zip
de8323486f3186ef8dae7b5f204b2bb785ac93970f50c23662220d43aae1fe6c  guix-build-e8f50c5debe2/output/arm64-apple-darwin/bitcoin-e8f50c5debe2-arm64-apple-darwin.tar.gz
93865368c5b9e6f88d273b64c85849c891444c598dcab872ee6ca37a4b6c814b  guix-build-e8f50c5debe2/output/dist-archive/bitcoin-e8f50c5debe2.tar.gz
d26be9fd3bef2b03459ac4de7fbe8c97b2bc8046313f53ff03add7d697c28403  guix-build-e8f50c5debe2/output/powerpc64-linux-gnu/SHA256SUMS.part
dfc44ce17b066576e337461d9843ca4b4a029b57add079a847a24a2a9e40321c  guix-build-e8f50c5debe2/output/powerpc64-linux-gnu/bitcoin-e8f50c5debe2-powerpc64-linux-gnu-debug.tar.gz
3a9fe2ca2b1f86855d9bd47414ae3ccd6652472b0e4510a73d5f24e2cb5d7a8d  guix-build-e8f50c5debe2/output/powerpc64-linux-gnu/bitcoin-e8f50c5debe2-powerpc64-linux-gnu.tar.gz
4d1a139f7e22fe3333c96b355fb5bb599cea0b0c7e83619fd2505a45f73c8e6e  guix-build-e8f50c5debe2/output/riscv64-linux-gnu/SHA256SUMS.part
15b148748fae1403ab79a00d9320b70f3c2275c492cc7761df41ee8d8be6a59a  guix-build-e8f50c5debe2/output/riscv64-linux-gnu/bitcoin-e8f50c5debe2-riscv64-linux-gnu-debug.tar.gz
f90f17f276d5cdd866aa07b5f853c1d110eb37df5b561b69d861b78678672719  guix-build-e8f50c5debe2/output/riscv64-linux-gnu/bitcoin-e8f50c5debe2-riscv64-linux-gnu.tar.gz
f8dfe9e92b52167c0bd21f57fd63b28882c9bf9673b038c38f39730eec0ad0a7  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/SHA256SUMS.part
7acc601c6d3b02ca757d10b2cd992cba6fa5c432cbd1c78f1f11bb11135c69ad  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/bitcoin-e8f50c5debe2-x86_64-apple-darwin-unsigned.tar.gz
ecc294be346a9755a51354a4cd0aeb33651506f306069a3966d67e7b45f44daa  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/bitcoin-e8f50c5debe2-x86_64-apple-darwin-unsigned.zip
c21ce456ec028304799c1cab06ac1dfc4c53d3add3f1f7dc4d5d42582a4bde95  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/bitcoin-e8f50c5debe2-x86_64-apple-darwin.tar.gz
b61b2ba603ec2830e351bd6631876e6f3098d022befa1a7564f4fb64f3fdf22b  guix-build-e8f50c5debe2/output/x86_64-linux-gnu/SHA256SUMS.part
5277cec1cdabacccd2dfae3607a71509074a58482861186fd71f11b7636380f7  guix-build-e8f50c5debe2/output/x86_64-linux-gnu/bitcoin-e8f50c5debe2-x86_64-linux-gnu-debug.tar.gz
c1b300217698837c4503e266c12dc10399f21774d919de5f875fe6f090da453f  guix-build-e8f50c5debe2/output/x86_64-linux-gnu/bitcoin-e8f50c5debe2-x86_64-linux-gnu.tar.gz
c49124e733de8587b95cbfc07076111e306de008c3f0390eb90a03076db23eaa  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/SHA256SUMS.part
a0d8a61b6a42beea561eb799825a2af43a1d07b9638c114887b90945e7a1fefc  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64-debug.zip
4a9915cf08aeb23775b5a7b9142b85cdce28686a97e05882f98777c4106b0e19  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64-setup-unsigned.exe
3d2ed27bf47823b48ba55a0839f8a064bf5dc9e59b2af25299de43663126394e  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64-unsigned.tar.gz
5c116eec993f8ce82cd0ca4e698570ac7d25542c4d17992562c736052ec72fde  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64.zip
```

**Review by sedited — APPROVED** [review]
Re-ACK e8f50c5debe2

Guix build (aarch64):
```
c3d79b98874ba44dbe6550fd18cdd78443ee1b4d0594dc1cc2e6b423dcded117  guix-build-e8f50c5debe2/output/aarch64-linux-gnu/SHA256SUMS.part
159a09c6ed25d498a48f3e3d53dd4c77b61c2ecbd824661593316030678ee5b0  guix-build-e8f50c5debe2/output/aarch64-linux-gnu/bitcoin-e8f50c5debe2-aarch64-linux-gnu-debug.tar.gz
34cd42b0a3574085c173073bbb12cbb19db2169efadab752335071eb897c242e  guix-build-e8f50c5debe2/output/aarch64-linux-gnu/bitcoin-e8f50c5debe2-aarch64-linux-gnu.tar.gz
0a62bc6b870d6dc36a4926ebcd546f001363ed3f5453ce80e20d929313e1a1ee  guix-build-e8f50c5debe2/output/arm-linux-gnueabihf/SHA256SUMS.part
374b3aeb02c3d9b0324ecce254ccdaf172e9711bf2ca5a821178cb57440f076c  guix-build-e8f50c5debe2/output/arm-linux-gnueabihf/bitcoin-e8f50c5debe2-arm-linux-gnueabihf-debug.tar.gz
dd088f2a79b31041f895ce3791ebaf4a9b6d79cefea79ec99dfbe1fee3adae87  guix-build-e8f50c5debe2/output/arm-linux-gnueabihf/bitcoin-e8f50c5debe2-arm-linux-gnueabihf.tar.gz
735bc9fd04d401527adc298d89b42a46c3d3a5dd80dea9a14ae00eefc6c1c5db  guix-build-e8f50c5debe2/output/arm64-apple-darwin/SHA256SUMS.part
4a478126accb0817e02c1009d0fa92dc2539fbd84b75fbc30d2eb97ae82c66a8  guix-build-e8f50c5debe2/output/arm64-apple-darwin/bitcoin-e8f50c5debe2-arm64-apple-darwin-unsigned.tar.gz
783ef9e80244cc9052c54bc4ece78d736f3109dc24f5fcef46c6b55a995398f4  guix-build-e8f50c5debe2/output/arm64-apple-darwin/bitcoin-e8f50c5debe2-arm64-apple-darwin-unsigned.zip
de8323486f3186ef8dae7b5f204b2bb785ac93970f50c23662220d43aae1fe6c  guix-build-e8f50c5debe2/output/arm64-apple-darwin/bitcoin-e8f50c5debe2-arm64-apple-darwin.tar.gz
93865368c5b9e6f88d273b64c85849c891444c598dcab872ee6ca37a4b6c814b  guix-build-e8f50c5debe2/output/dist-archive/bitcoin-e8f50c5debe2.tar.gz
d26be9fd3bef2b03459ac4de7fbe8c97b2bc8046313f53ff03add7d697c28403  guix-build-e8f50c5debe2/output/powerpc64-linux-gnu/SHA256SUMS.part
dfc44ce17b066576e337461d9843ca4b4a029b57add079a847a24a2a9e40321c  guix-build-e8f50c5debe2/output/powerpc64-linux-gnu/bitcoin-e8f50c5debe2-powerpc64-linux-gnu-debug.tar.gz
3a9fe2ca2b1f86855d9bd47414ae3ccd6652472b0e4510a73d5f24e2cb5d7a8d  guix-build-e8f50c5debe2/output/powerpc64-linux-gnu/bitcoin-e8f50c5debe2-powerpc64-linux-gnu.tar.gz
4d1a139f7e22fe3333c96b355fb5bb599cea0b0c7e83619fd2505a45f73c8e6e  guix-build-e8f50c5debe2/output/riscv64-linux-gnu/SHA256SUMS.part
15b148748fae1403ab79a00d9320b70f3c2275c492cc7761df41ee8d8be6a59a  guix-build-e8f50c5debe2/output/riscv64-linux-gnu/bitcoin-e8f50c5debe2-riscv64-linux-gnu-debug.tar.gz
f90f17f276d5cdd866aa07b5f853c1d110eb37df5b561b69d861b78678672719  guix-build-e8f50c5debe2/output/riscv64-linux-gnu/bitcoin-e8f50c5debe2-riscv64-linux-gnu.tar.gz
f8dfe9e92b52167c0bd21f57fd63b28882c9bf9673b038c38f39730eec0ad0a7  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/SHA256SUMS.part
7acc601c6d3b02ca757d10b2cd992cba6fa5c432cbd1c78f1f11bb11135c69ad  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/bitcoin-e8f50c5debe2-x86_64-apple-darwin-unsigned.tar.gz
ecc294be346a9755a51354a4cd0aeb33651506f306069a3966d67e7b45f44daa  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/bitcoin-e8f50c5debe2-x86_64-apple-darwin-unsigned.zip
c21ce456ec028304799c1cab06ac1dfc4c53d3add3f1f7dc4d5d42582a4bde95  guix-build-e8f50c5debe2/output/x86_64-apple-darwin/bitcoin-e8f50c5debe2-x86_64-apple-darwin.tar.gz
b61b2ba603ec2830e351bd6631876e6f3098d022befa1a7564f4fb64f3fdf22b  guix-build-e8f50c5debe2/output/x86_64-linux-gnu/SHA256SUMS.part
5277cec1cdabacccd2dfae3607a71509074a58482861186fd71f11b7636380f7  guix-build-e8f50c5debe2/output/x86_64-linux-gnu/bitcoin-e8f50c5debe2-x86_64-linux-gnu-debug.tar.gz
c1b300217698837c4503e266c12dc10399f21774d919de5f875fe6f090da453f  guix-build-e8f50c5debe2/output/x86_64-linux-gnu/bitcoin-e8f50c5debe2-x86_64-linux-gnu.tar.gz
c49124e733de8587b95cbfc07076111e306de008c3f0390eb90a03076db23eaa  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/SHA256SUMS.part
a0d8a61b6a42beea561eb799825a2af43a1d07b9638c114887b90945e7a1fefc  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64-debug.zip
4a9915cf08aeb23775b5a7b9142b85cdce28686a97e05882f98777c4106b0e19  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64-setup-unsigned.exe
3d2ed27bf47823b48ba55a0839f8a064bf5dc9e59b2af25299de43663126394e  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64-unsigned.tar.gz
5c116eec993f8ce82cd0ca4e698570ac7d25542c4d17992562c736052ec72fde  guix-build-e8f50c5debe2/output/x86_64-w64-mingw32/bitcoin-e8f50c5debe2-win64.zip
```

**Pull request merged**